### PR TITLE
Utility to update entry properties: update-entry-properties

### PR DESCRIPTION
**Pull request by frenchy64**
This is what I came up with to answer this slack question: https://clojurians.slack.com/archives/CLDK6MFMK/p1712696483105789

### CODE REVIEW

File: src/malli/util.cljc [hunk 1]
```diff
@@ -111,6 +111,17 @@
   (let [schema (m/schema ?schema)]
     (apply m/-update-properties schema f args)))
 
+(defn update-entry-properties
+  "Returns a Schema instance with updated properties for entry k."
+  [?schema k f & args]
+  (let [schema (m/schema ?schema)
+        e (or (find schema k)
+              (m/-fail! ::no-entry {:schema schema :k k}))
+        [k p v] (case (count e)
```
Comment (ikitommi) [inline src/malli/util.cljc:120]: entry contains always 3 elements I believe.

File: test/malli/util_test.cljc [hunk 2]
```diff
@@ -990,3 +990,25 @@
                  (mu/assoc-in [:foo :bar] :int)
                  (mu/assoc-in [:foo :baz] :int)
                  (mu/closed-schema)))))
+
+(deftest update-entry-properties-test
+  (is (= [:map [:me {:a 1, :b 1} :int]]
```
Comment (ikitommi) [inline test/malli/util_test.cljc:995]: please add a failing test here too.

**Review by ikitommi — COMMENTED** [review]
I think the implementation can be simplified a bit

**Comment by ikitommi** [comment]
well, I can do those. merging. thanks!

**Comment by ikitommi** [comment]
https://github.com/metosin/malli/commit/48544d533bf700151c886c13064c29a916a69fa9?w=1

**Comment by ikitommi** [comment]
```clojure
(mu/find [:map [:x :int]] :x) 
; => [:x nil :int]
```

**Comment by frenchy64** [comment]
Thanks! I was trying to reverse engineer this line, tbh I still don't understand it. https://github.com/metosin/malli/blob/48544d533bf700151c886c13064c29a916a69fa9/src/malli/core.cljc#L360

**Comment by ikitommi** [comment]
`mu/find` wraps the search key into a vector of `[::m/find key]`, `m/-get-entries` unwraps it and returns the full entry instead of just the value behind the key. Entries are still always of size 3.

```clojure
(m/-get-entries (m/schema [:map [:x :int]]) :x nil)
; => :int

(m/-get-entries (m/schema [:map [:x :int]]) [::m/find :x] nil)
; => [:x nil :int]
```

**Pull request merged**